http-client.ts

```diff
@@ -229,7 +237,10 @@ export class HttpClient implements Server.IHttpClient {
 	}
 
 	private getErrorMessage(response: Server.IRequestResponseData, body: string): string {
-		if (response.statusCode === 402) {
+		if (response.statusCode === 407) {
```

Comment:
I'd suggest we add some constants with the frequently-used Http Status code for easier readability